app/src/main/java/mozilla/lockbox/presenter/OnboardingFingerprintAuthPresenter.kt

```diff
@@ -74,4 +77,13 @@ class OnboardingFingerprintAuthPresenter(
             is AuthenticationState.Error -> view.onError(state.error)
         }
     }
+
+    private fun triggerNextOnboarding() {
+        if (SettingStore.shared.autofillAvailable) {
```

Comment:
😬 ok, I think this one is my fault...

the `get()` function for `autofillAvailable` is annotated with `@RequiresApi(Build.VERSION_CODES.O)`, but that isn't enough to force the compiler to warn you about it unfortunately. Rather than checking the `SettingStore.shared.autofillAvailable` initially, you need to check your version code _first_ and then whether autofill is available. This code will make the app crash on versions < 26.

Reply:
Oh cool! Makes sense to me 👍